Dockerfile.ci

```diff
@@ -0,0 +1,18 @@
+ARG commit=latest
+FROM elifesciences/hypothesis-dummy:${commit}
+
+# extract into another image and put here with multi-stage build
+USER root
+RUN git clone https://github.com/asm89/smoke.sh /opt/smoke.sh
+USER www-data
+
+# extract into another image and put here with multi-stage build
+RUN composer global --no-interaction require elife/proofreader-php=dev-master
+ENV PATH="/var/www/.composer/vendor/bin:${PATH}"
```

Comment:
Should be part of the Composer install?

Reply:
This path is only used here because normally composer is run by `elife` so that the files that it creates are not writable by `www-data`. I'll redo this installation in a separate `proofreader` image to avoid these issues.